pkg/cluster/provider.go

```diff
@@ -105,6 +106,27 @@ func (p *Provider) Delete(name, explicitKubeconfigPath string) error {
 	return internaldelete.Cluster(p.logger, p.ic(name), explicitKubeconfigPath)
 }
 
+// Deletes all clusters created by getting list of cluster names and using Delete function.
+func (p *Provider) DeleteAll(explicitKubeconfigPath string) error {
+	clusters, err := p.provider.ListClusters()
+	if err != nil {
+		return err
+	}
+
+	if len(clusters) == 0 {
+		return errors.New("No kind clusters found.")
+	}
+
+	for _, cluster := range clusters {
+		err = internaldelete.Cluster(p.logger, p.ic(cluster), explicitKubeconfigPath)
+		if err != nil {
+			return err
```

Comment:
would the error string include the name of the cluster that failed to delete?
[2]

Reply:
I think the answer to this is no. I think, as of now, the error for kind does not include the name. Would you like me to add it in or keep the current behavior? I agree that it would be good to include it.

Reply:
can be added here (on the side of the caller) using `errors.Wrap(...`

Reply:
Thanks! Pushed something up.